streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -41,8 +41,8 @@
 import org.apache.kafka.common.utils.Time;
 import org.apache.kafka.streams.errors.LogAndContinueExceptionHandler;
 import org.apache.kafka.streams.errors.TopologyException;
-import org.apache.kafka.streams.internals.KeyValueStoreFacade;
-import org.apache.kafka.streams.internals.WindowStoreFacade;
+import org.apache.kafka.streams.test.internal.KeyValueStoreFacade;
+import org.apache.kafka.streams.test.internal.WindowStoreFacade;
```

Comment:
Javadoc is unable to generate docs for this (TopologyTestDriver) file because it depends on these classes, which are also in test-utils, but are excluded in the build.gradle spec.

I was unable to override the exclusion with a more specific inclusion, so instead I just moved these classes to a different package that matches only the "include" patterns in `:streams:test-utils:javadoc`.

Reply:
Have you checked that the produced javadoc does not include these internal files?

Reply:
I didn't before, but I did just check on my latest strategy, and they are not generated.